src/yaourt.sh.in

```diff
@@ -127,6 +128,15 @@ yaourt_install_packages() {
 	prepare_status_list
 	SP_ARG="" sync_packages "${ARGS[@]}" || return $?
 	analyse_status_list
+
+	# Trigger Backup
```

Comment:
The user is asked to install `gist` each time a package is installed. IMHO, it will be annoying.